Myanmar/HaveFun.md

```diff
@@ -0,0 +1,17 @@
+I was trying to type in Myanmar. However, the text editor I was using doesn't seem to support Myanmar Font.
+Anyway, it has been a journey and I am proud that I have come so far learning git and github.
+Before this online class, I had no idea about what "git" and "github" are.
+I have seen them quite a lot in the news and I wondered what are they?
+The day has finally arrived! I have finished the course. Yay~~
+
+I don't know what other people write about. Honestly, I read one and it was about Jesus and love. 
+Well, at least it is better than pornographic stuffs. 
+
+I had no idea about the source file when I first clone it to my repo. Then I saw folders, named accordingly to their country and under each folder, there is(are) folder(s) named after the topics of the letter. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.